connector_ecommerce/product.py

```diff
@@ -115,13 +115,13 @@ def _price_changed(self, vals):
         There is no guarantee that's the price actually changed,
         because it depends on the pricelists.
         """
-        type_model = self.env['product.price.type']
-        price_fields = type_model.sale_price_fields()
-        if any(field in price_fields for field in vals):
-            session = ConnectorSession(self.env.cr, self.env.uid,
-                                       context=self.env.context)
-            for prod_id in self.ids:
-                on_product_price_changed.fire(session, self._name, prod_id)
+#         type_model = self.env['product.price.type']
+#         price_fields = type_model.sale_price_fields()
+#         if any(field in price_fields for field in vals):
+        session = ConnectorSession(self.env.cr, self.env.uid,
+                                   context=self.env.context)
+        for prod_id in self.ids:
```

Comment:
It is fired on every write, it should be called only when a price is actually changed